Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.12.3-slim
+LABEL maintainer="94nj111@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=True
+ENV PYTHONUNBUFFERED=True
+
+WORKDIR /app
+
+ARG UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/app" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    appuser
+
+RUN pip install --no-cache-dir requests
```

Comment:
It's recommended to install all required Python packages from a `requirements.txt` file instead of installing them individually. This makes dependency management easier and more scalable.